deploy/cloud/helm/platform/components/operator/values.yaml

```
@@ -98,6 +95,9 @@ dynamo:
     password: ""
     existingSecretName: ''
     secure: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)